comps/intent_detection/langchain/intent_detection.py

```diff
@@ -0,0 +1,47 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+
+from langchain import LLMChain, PromptTemplate
+from langchain_community.llms import HuggingFaceEndpoint
+from langsmith import traceable
+
+from comps import GeneratedDoc, LLMParamsDoc, ServiceType, opea_microservices, register_microservice
+from comps.intent_detection.langchain.template import IntentTemplate
+
+
+@register_microservice(
+    name="opea_service@llm_intent",
+    service_type=ServiceType.LLM,
+    endpoint="/v1/chat/intent",
+    host="0.0.0.0",
+    port=9000,
```

Comment:
don't use port 9000 since it's occupied by llm microservice.

Reply:
which port do you recommend? I will change.